tecken/base/symboldownloader.py

```diff
@@ -159,7 +153,7 @@ def _get(self, symbol, debugid, filename, refresh_cache=False):
                 source.base_url, self.make_url_path(prefix, symbol, debugid, filename)
             )
             logger.debug(f"Looking for symbol file by URL {file_url!r}")
-            if last_modified := get_last_modified(file_url, _refresh=refresh_cache):
+            if last_modified := get_last_modified(file_url):
```

Comment:
r+ under the assumption that `refresh_cache` arg throughout this file will be removed in a later update

Reply:
Thanks! I've removed them.